modules/ROOT/pages/update-acm.adoc

```diff
@@ -21,6 +21,9 @@ The staging environment will have a partial or full copy of the data in the prod
 
 To update your API community:
 
+. Before performing any steps, read all the steps, including pre-install and post-install.
++
+Some steps may require mandatory downtime during the update.
```

Comment:
@wwarner-salesforce we should change this, Jesus told me to put something like this: 

potential degraded service of your community: APIs, docs and client apps pages may not work after the upgrade until you perform the required config changes. Notify your customers and stakeholders as you see fit.

Reply:
Updated: "If necessary, notify your customers and stakeholders of the update, and that APIs, docs, and client app pages may not work during the update during the time after installing the new package and before completing the required configuration changes."